charts/app-of-apps/templates/ckan-application.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   source:
     repoURL: https://github.com/alphagov/govuk-dgu-charts
     path: charts/ckan
-    targetRevision: HEAD
+    targetRevision: dj-maisy/dgu-arm-integration
```

Comment:
With the `targetRevision` key set to a branch, doesn't this need to be cherry-picked into main/HEAD before it'll do anything?

Reply:
the commit for the targetRevision will be dropped before approving the PR